README.md

```diff
@@ -58,15 +59,30 @@ If this command works properly, you will be redirected to an auth route in the G
 
 # Further Details: Backend
 
-## Conda Env Setup
+## Poetry Env Setup
 
-- `conda env create -f environment.yml` in the `/conda` directory
+- `poetry install` in the project root directory
 
-- Updating an environment: `conda env update -f environment.yml` in the `/conda` directory
+- Updating dependencies: `poetry update`
+
+# pyenv setup
+- To install pyenv, a python version management tool, you can use the following command via homebrew: `brew install pyenv`
+
+- Installing python version: `pyenv install 3.9.17`
+
+- Set the global python version: `pyenv global 3.9.17`
+
+- Verify the installation using `pyenv --version`
+
+- pyenv version control: `pyenv install -v 3.9.17`
```

Comment:
what does this command do exactly? we should make it more clear to someone new to run commands 1, 2, 3 in step by step fashion

Reply:
I thought i gave it in a step by step format though, and at the end gave them command for pyenv version control and command if they run into any issues. Lmk how i can restructure it.

Reply:
i think you should try to make it clear that for getting python 3.9.17 in pyenv, run the following commands. is the -v thing needed? what is the advantage you get?

Reply:
I was just showing the command for version control there for reference. I can remove it.

Reply:
if its not very useful, then lets remove that part. i think brevity + clarity matter. a lot in a good readme.md

Reply:
good point